CONTRIBUTING.md

```diff
@@ -121,12 +121,14 @@ See more about TypeORM migrations at official docs [Migrations](https://github.c
 
 ## Pull Requests
 
-- Check how to create a [Pull Request](https://help.github.com/articles/creating-a-pull-request/)
-- Send a pull request to `master` branch
-- Fill template
-- Write a meaningful description
-- Include screenshots and animated GIFs in your pull request whenever possible
-- Add `deploy` label if you want to test your changes
+- Check how to create a [Pull Request](https://help.github.com/articles/creating-a-pull-request/) (PR).
+- PR titles must follow [Conventional Commits](#git-commit-messages).
+- Do not include issue IDs in the PR title.
```

Comment:
Btw I've been always curious why :)

Reply:
you get PR Id link in commit message when you merge and I want to avoid confusion. PR link is enough to get trace to issue

<img width="664" alt="image" src="https://github.com/user-attachments/assets/d48122f5-6d97-47c2-b3f5-55e6cdfbf342" />